data/human/intro missions.txt

```diff
@@ -145,7 +145,7 @@ mission "Intro [2 Transport]"
 				`	"Sure!"`
 				`	"No thanks, I think I'll strike off on my own now."`
 					decline
-			`	"Great," he says. "This is Chuck, and Sarah, and their son Carl. Let's help them carry their luggage to your ship." As you begin walking back, slightly ahead of them, James whispers to you, "Tourists are always a good way to make money. They're on vacation, so they don't mind paying a bit extra. But no point in your cargo space going to waste, so take a look at the job board and see if there are any delivery missions we can run at the same time. And if you want to be a little safer should we come across any pirates, sell off any guns you might have. It might seem counterintuitive, but being unarmed makes you less of a target to pirates if there are others in the system that they could be defending themselves against."`
+			`	"Great," he says. "This is Chuck, and Sarah, and their son Carl. Let's help them carry their luggage to your ship." As you begin walking back, slightly ahead of them, James whispers to you, "Tourists are always a good way to make money. They're on vacation, so they don't mind paying a bit extra. But no point in your cargo space going to waste, so take a look at the job board and see if there are any delivery missions we can run at the same time. And if you want to be a little safer should we come across any pirates, sell off any damaging weapons you might have. It might seem counterintuitive, but being unarmed makes you less of a target to pirates if there are others in the system that they could be defending themselves against."`
```

Comment:
This seems more confusing to me. The phrase "damaging weapon" implies the existence of non-damaging weapons. What exactly is a non-damaging weapon? If it's non-damagimg, then is it even a weapon? Anti-missile turrets destroy (i.e. damage) missiles, so might someone think that AM turrets still need to be sold off? Having someone in-universe say "damaging weapon" also feels like a very forced way to try and exclude AM turrets and tractor beams. James is here to explain tips for surviving in-universe, not explain gameplay mechanics. If the pirates don't care about AM turrets, have James explicitly explain that.

Reply:
~~thats what I had before QuantumShark changed it~~
also its not only AM, its also tractor beams now

Reply:
Offensive weapons, maybe?

Reply:
I'm down with anything thats clearer than the "gun" we had

Reply:
I honestly don't see a reason to change this line at all. This is the transport string. You're in a shuttle that can only possibly have a single gun. It makes sense for James to refer specifically to selling off guns in this moment. But the very next sentence expands upon why you should do that, stating that pirates will focus more on defending themselves against others rather than going for an unarmed target. That is, if you can't shoot at the pirates, they're less likely to shoot at you. You can't shoot at pirates with AM turrets or tractor beams. Therefore, there's nothing about this that says that an AM turret or tractor beam would make you a target.

If we were to change it, I think that saying offensive weapons would be the best route. AM turrets are explicitly defensive, and tractor beams aren't really weapons, so saying offensive weapons should make it clear that those two are fine to have.
```suggestion
			`	"Great," he says. "This is Chuck, and Sarah, and their son Carl. Let's help them carry their luggage to your ship." As you begin walking back, slightly ahead of them, James whispers to you, "Tourists are always a good way to make money. They're on vacation, so they don't mind paying a bit extra. But no point in your cargo space going to waste, so take a look at the job board and see if there are any delivery missions we can run at the same time. And if you want to be a little safer should we come across any pirates, sell off any offensive weapons you might have. It might seem counterintuitive, but being unarmed makes you less of a target to pirates if there are others in the system that they could be defending themselves against."`
```

Reply:
```suggestion
			`	"Great," he says. "This is Chuck, and Sarah, and their son Carl. Let's help them carry their luggage to your ship." As you begin walking back, slightly ahead of them, James whispers to you, "Tourists are always a good way to make money. They're on vacation, so they don't mind paying a bit extra. But no point in your cargo space going to waste, so take a look at the job board and see if there are any delivery missions we can run at the same time. And if you want to be a little safer should we come across any pirates, sell off any offensive weapons you might have. It might seem counterintuitive, but being unarmed makes you less of a target to pirates if there are others in the system that they could be defending themselves against."`
```